### PR TITLE
 Answer:1

### PR DESCRIPTION
**Pull request by AshishKumar409**

### CODE REVIEW

File: apps/angular/projection/src/app/ui/list-item/list-item.component.ts [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-    <div class="border border-grey-300 py-1 px-2 flex justify-between">
-      {{ name }}
-      <button (click)="delete(id)">
-        <img class="h-5" src="assets/svg/trash.svg" />
-      </button>
-    </div>
+    <ng-content />
+    <button (click)="delete.emit()">
+      <img class="h-5" src="assets/svg/trash.svg" />
+    </button>
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/list-item/list-item.component.ts:12]: 👍

File: apps/angular/projection/src/app/ui/card/card.component.html [hunk 2]
```diff
@@ -1,27 +1,7 @@
...
-    Add
-  </button>
-</div>
+<ng-content select="img" />
+<section>
+  <ng-container *ngFor="let item of list">
+    <ng-container [ngTemplateOutlet]="template" [ngTemplateOutletContext]="{$implicit:item}" />
+  </ng-container>
+</section>
+<ng-content select="button" />
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/card/card.component.html:7]: 👍

File: apps/angular/projection/src/app/ui/card/card.component.ts [hunk 3]
```diff
@@ -1,34 +1,28 @@
...
 })
-export class CardComponent {
-  @Input() list: any[] | null = null;
-  @Input() type!: CardType;
-  @Input() customClass = '';
+export class CardComponent<T> {
+  @Input() list: T[] | null = null;
+  @ContentChild('rowRef', { read: TemplateRef }) template!: TemplateRef<{
+    $implicit: T;
+  }>;
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/card/card.component.ts:25]: improvement: avoid magic string, use directives

File: apps/angular/projection/src/app/component/city-card/city-card.component.ts [hunk 4]
```diff
@@ -1,13 +1,56 @@
...
+      .bg-light-orange {
+        background-color: rgb(255, 230, 204);
+      }
+    `,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: Observable<City[]> = this.store.cities$;
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/component/city-card/city-card.component.ts:42]: improvement: use signals 😜

**Comment by tomalaforge** [comment]
great work. Nothing much to say 👍

File: apps/angular/projection/src/app/component/city-card/city-card.component.ts (hunk 4 above)
Comment (AshishKumar409) [reply apps/angular/projection/src/app/component/city-card/city-card.component.ts:42]: Thank you for the review😃, will try to implement this with signals.

**Pull request closed without merging**